modules/recurlyjs/recurlyjs.module

```diff
@@ -121,3 +122,17 @@ function recurlyjs_settings_form_submit(array &$form, FormStateInterface $form_s
     ->set('recurlyjs_accept_paypal', $form_state->getValue('recurlyjs_accept_paypal'))
     ->save();
 }
+
+/**
+ * Implements hook_entity_type_alter().
+ */
+function recurlyjs_entity_type_alter(array &$entity_types) {
+  $recurly_entity_type = \Drupal::config('recurly.settings')->get('recurly_entity_type') ?: 'user';
```

Comment:
I keep seeing this same line of code with the same default value and it makes me think we need to store the default value elsewhere or make this into a helper function or method.